src/components/SiteHeader/SiteHeader.jsx

```diff
@@ -16,20 +16,47 @@ const HEADER_HIDDEN = {
 /**
  * JSX element for Barnes logos.
  */
-const Logo = ({ size, width, height }) => (
-  <svg className={`a-logo__svg a-logo__svg--${size}`} width={width} height={height} aria-labelledby='logo-title'>
+const Logo = ({ size, width, height, logo, className }) => (
+  <svg className={`a-logo__svg a-logo__svg--${size} ${className ? className : ""}${size === 's' ? "H" : ""}`} width={width} height={height} aria-labelledby='logo-title'>
     <title id='logo-title'>Barnes</title>
-    <use xlinkHref={`#icon--logo-${size}`}></use>
+    <use xlinkHref={`#icon--logo-${size === 's' ? "100H" : logo ? "100" : size}`}></use>
   </svg>
 );
 
+
 // Static info mapped to static Logo JSX element.
-const LOGOS = [
+const logoSizes = [
   { size: 's', width: 121, height: 37 },
   { size: 'm', width: 146, height: 45 },
   { size: 'l', width: 164.958, height: 50 },
   { size: 'xl', width: 200, height: 62 },
-].map((logo => <Logo key={logo.size} {...logo} />));
+];
+
+// keep width constant, scale height
+const logo100Sizes = [
+  { size: 's', width: 263.66, height: 37 },
+  { size: 'm', width: 146, height: 70.32 },
+  { size: 'l', width: 164.958, height: 79.45 },
+  { size: 'xl', width: 200, height: 95.75 },
+];
+
+
+
+const Logos = ({ ref, tabIndex, isCentennial}) => (
+  <a
+    className='a-logo g-header__logo'
+    href={MAIN_WEBSITE_DOMAIN}
+    tabIndex={tabIndex}
+    ref={ref}
+  >
+    {isCentennial ? (
+      logo100Sizes.map((logo => <Logo key={logo.size} {...logo} logo={100} height={undefined} className="a-logo__svg--100" />))
+    ) : (
+      logoSizes.map((logo => <Logo key={logo.size} {...logo} />))
+    )}
```

Comment:
And finally, we can simplify this as well since we included all of the attributes in each logo size object:
```
{isCentennial ? (
      logo100Sizes.map((logo => <Logo key={logo.size} {...logo} className="a-logo__svg--100" />))
    ) : (
      logoSizes.map((logo => <Logo key={logo.size} {...logo} />))
    )}
```